doc/vector/insns/vclmaccgh.adoc

```diff
@@ -141,15 +142,17 @@ function clause execute (VCLMULGF(vs2, vs1, vd)) = {
     let H = brev(get_velem(vs1,EGW=128,i)); // Hash subkey
     let X = brev(get_velem(vs2,EGW=128,i)); // block cipher output
 
-    let sum = Y ^ X;
 
-    let prod[254:0]   = clmul_128(sum, H);
+    let prod[254:0]   = clmul_128(Y, H);
       // Reduce product by irreducible polynomial
     let red1 = clmul_13(prod[254:249], 0x87);
     let prod[133:121] ^= red1;
     let red2 = clmul_121(prod[248:128],0x87);
     let prod[127:0] ^= red2;
-    let result = brev(prod[127:0]);  // bit reverse in byte to get back to GCM standard ordering
+
+    let sum[127:0] = prod ^ X;
+
+    let result = brev(sum[127:0]);  // bit reverse in byte to get back to GCM standard ordering
```

Comment:
With this scheme I think we can save the `brev` of `X` and post-pone the final `xor` after `brev`-ing back the product.